tests/lib/devicetree/api/src/main.c

```diff
@@ -1537,6 +1542,9 @@ static void test_enums_required_false(void)
 	zassert_equal(DT_ENUM_IDX_OR(DT_NODELABEL(test_enum_int_default_1),
 				     val, 4),
 		      4, "");
+	/* DT_INST_ENUM_IDX_OR (using vnd,enum-required-false-holder) */
```

Comment:
Can you add an explicit test for `DT_INST_ENUM_IDX`.

Reply:
@galak it's tested in `test_enums`, do you mean adding a new test for `INST` based macros?

Reply:
I mean add a test for `INST`.  I don't want to assume that DT_INST_ENUM_IDX_OR is implemented using DT_INST_ENUM_IDX

Reply:
isn't that sufficient https://github.com/zephyrproject-rtos/zephyr/pull/40423/commits/287db6e46b1a3b6ac75bc54f5f9a99234df28b3c#diff-d3ce6b001979843da3e07890fd4bdbf9008ee4fb2c813c95baf108be1ec2b539R1521 ?

Reply:
ah, I see.